source/_integrations/arcam_fmj.markdown

```diff
@@ -41,15 +41,19 @@ Supported devices:
 - SA 30
 - SR 250
 
-This integration may also work with JBL and AudioControl receivers and processors as they share the same firmware as Arcam.
+This integration may also work with [JBL](https://www.jblsynthesis.com/products/electronics/) and [AudioControl](https://www.audiocontrol.com/home-audio/) receivers and processors as they share the same firmware as Arcam.
 
 - SDP-55/58
 
 {% include integrations/config_flow.md %}
 
 ## Power state
 
-Arcam FMJ receivers turn off their network port when in standby, the component will try to reconnect to the receiver every 5 seconds. This means powering on the first zone is not possible over the built-in network connection. Two options for complete power control exists: IR or Serial gateway.
+Arcam receivers turn off their network port when in standby, the component will try to reconnect to the receiver every 5 seconds. This means powering on the first zone is not possible over the built-in network connection. 
+
+Note: Some newer models offer the ability to configure the device to keep the network port active when in standby mode. This can be found under `HDMI Settings` -> `HDMI Bypass & IP`. Enabling `HDMI & IP On` will allow full power control from Home Assistant.
```

Comment:
```suggestion
Note: Some newer models offer the ability to configure the device to keep the network port active when in standby mode. This can be found under **HDMI Settings** > **HDMI Bypass & IP**. Enabling **HDMI & IP On** will allow full power control from Home Assistant.
```
Changed these to bold because we are using the MS Style Guide.
<https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui>